packet_forwarder/global_conf.json.sx1250.AS923-2

```diff
@@ -0,0 +1,75 @@
+{
+  "SX130x_conf": {
+    "spidev_path": "/dev/spidev0.0",
+    "lorawan_public": true,
+    "clksrc": 0,
+    "antenna_gain": 0,
+    "full_duplex": false,
+    "precision_timestamp": {
+      "enable": false,
+      "max_ts_metrics": 255,
+      "nb_symbols": 1
+    },
+    "radio_0": {
+      "enable": true,
+      "type": "SX1250",
+      "freq": 921700000,
+      "rssi_offset": -215.4,
+      "rssi_tcomp": {"coeff_a": 0, "coeff_b": 0, "coeff_c": 20.41, "coeff_d": 2162.56, "coeff_e": 0},
+      "tx_enable": true,
+      "tx_freq_min": 921300000,
+      "tx_freq_max": 922900000,
+      "tx_gain_lut":[
+          {"rf_power": 12, "pa_gain": 1, "pwr_idx": 4},
+          {"rf_power": 13, "pa_gain": 1, "pwr_idx": 5},
+          {"rf_power": 14, "pa_gain": 1, "pwr_idx": 6},
+          {"rf_power": 15, "pa_gain": 1, "pwr_idx": 7},
+          {"rf_power": 16, "pa_gain": 1, "pwr_idx": 8},
+          {"rf_power": 17, "pa_gain": 1, "pwr_idx": 9},
+          {"rf_power": 18, "pa_gain": 1, "pwr_idx": 10},
+          {"rf_power": 19, "pa_gain": 1, "pwr_idx": 11},
+          {"rf_power": 20, "pa_gain": 1, "pwr_idx": 12},
+          {"rf_power": 21, "pa_gain": 1, "pwr_idx": 13},
+          {"rf_power": 22, "pa_gain": 1, "pwr_idx": 14},
+          {"rf_power": 23, "pa_gain": 1, "pwr_idx": 15},
+          {"rf_power": 24, "pa_gain": 1, "pwr_idx": 16},
+          {"rf_power": 25, "pa_gain": 1, "pwr_idx": 17},
+          {"rf_power": 26, "pa_gain": 1, "pwr_idx": 19},
+          {"rf_power": 27, "pa_gain": 1, "pwr_idx": 20}
+      ]
+    },
+    "radio_1": {
+      "enable": true,
+      "type": "SX1250",
+      "freq": 922500000,
+      "rssi_offset": -215.4,
+      "rssi_tcomp": {"coeff_a": 0, "coeff_b": 0, "coeff_c": 20.41, "coeff_d": 2162.56, "coeff_e": 0},
+      "tx_enable": false
+    },
+     "chan_multiSF_All": {"spreading_factor_enable": [ 5, 6, 7, 8, 9, 10, 11, 12 ]},
```

Comment:
I confirmed that this doesn't break older packet forwarder versions. I loaded this config on a Rak hotspot and it started up just fine.

Reply:
Oh good to hear. I nabbed this from the sx1302_hal master because I liked their new formatting for the channel configs, but I didn't realize I took a whole new field.